.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
+  "plugins": ["patternfly-react"],
+  "extends": [
+    "plugin:patternfly-react/recommended",
+    "./node_modules/@theforeman/vendor-dev/eslint.extends.js",
+  ],
+  "rules": {
+    "prettier/prettier": ["error", {
+      "singleQuote": true,
+      "trailingComma": "es5"
+    }],
+    "import/no-unresolved": ["error", {
+      "ignore": ['foremanReact/.*']
+    }],
+    "import/extensions": ["error", {
+      "ignore": ['foremanReact/.*']
+    }],
+  },
+  "globals": {
+    "document": false,
+    "escape": false,
+    "navigator": false,
+    "unescape": false,
+    "window": false,
+    "$": true,
+    "_": true,
+    "__": true,
+    "n__": true,
+    "d3": true
+  }
+}
```

Comment:
imo it could be shifted now to:
`
{
  "plugins": ["@theforeman/foreman"],
  "extends": [
    "plugin:@theforeman/foreman/core",
    "plugin:@theforeman/foreman/plugins"
  ]
}
`